rpc/examples/echo/node-client/src/index.ts

```diff
@@ -0,0 +1,108 @@
+import type { PartialMessage } from "@bufbuild/protobuf";
+import { ConnectError, createPromiseClient, PromiseClient } from "@connectrpc/connect";
+import { createWritableIterable } from "@connectrpc/connect/protocol";
+import { dialDirect, dialWebRTC } from "@viamrobotics/rpc";
+import { EchoService } from "./gen/proto/rpc/examples/echo/v1/echo_connect.js";
+import { EchoBiDiRequest, EchoMultipleRequest, EchoRequest } from "./gen/proto/rpc/examples/echo/v1/echo_pb.js";
+
+import { createGrpcTransport } from '@connectrpc/connect-node';
+import wrtc from "node-datachannel/polyfill";
+global.window = global.window || {};
+window.VIAM = {
+	GRPC_TRANSPORT_FACTORY: (opts: any) => createGrpcTransport({ httpVersion: "2", ...opts }),
+};
```

Comment:
it's more for the isomorphism. there's likely a better pattern but I don't want to spend the cycles on that right now since it's for lesser used use cases

Reply:
will do globalThis